lua/neo-tree/setup/init.lua

```diff
@@ -297,7 +297,7 @@ local function merge_global_components_config(components, config)
   return merged_components
 end
 
-local merge_renderers = function (default_config, source_default_config, user_config)
+local merge_renderers = function(default_config, source_default_config, user_config)
```

Comment:
This change was just sumneko-lua auto-formatting the code. I suppose it is ok to keep it?

Reply:
That's as it should be. I'm not sure how these formatting issues sneak in, but it should be without the space.